src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println("Hello there! I'm Duke.\nWhat can I do for you?\n");
+
+        Task[] taskList = new Task[100];
+        int count = 0;
+        Scanner sc = new Scanner(System.in);
+        Boolean isSame = true;
+
+        while (isSame) {
+            String input = sc.nextLine();
+            String stringTask;
+            String stringDate = null;
+
+            /*Processes the input of the user.
+            The type of task is indicated by the first substring and the task description is found after the first blank space.
+            If a forward slash present,  ths substring is split into the task and the end date which is indicated by the slash.
+             */
+            int indexOfSpace = input.indexOf(" ");
+            String subString = input.substring(indexOfSpace+1);
+
+            if (subString.contains("/")) {
+                int indexOfSlash = subString.indexOf("/");
+                stringTask = subString.substring(0, indexOfSlash-1);
+                String subStringDate = subString.substring(indexOfSlash);
+                int indexNext = subStringDate.indexOf(" ");
+                String subStringEnd = subStringDate.substring(indexNext+1);
+                stringDate = subStringEnd;
+            }
```

Comment:
If I recall correctly, if-else statements should have the form
```
if (condition) {
    statements;
} else {
    statements;
}
```
Perhaps you could review the java coding standard again?